### PR TITLE
Ktaschek sponsor card

### LINKED ISSUE #24: Create sponsor card
**Description**
Refer to admin v5 figma wireframe and create the sponsor cards. The sponsor card is just the single square containing the sponsor name and info. 

**Acceptance Criteria**

Steps:

- Create profile picture placeholder
- Add sponsor/organization name
- Add sponsor user
- Add sponsor user email
- Add w-9 label (no need to link to anything just the label is fine for now)

### PR DESCRIPTION
**Pull request by ktaschek**
## Developer: Kyle Taschek

Closes #24 

### Pull Request Summary

Complete Issue 24: Create Sponsor Card with layout and images (no color)

### Modifications

{list out the files created/modified and a brief description of what was changed}
- `/components/sponsor_card.tsx`: Created sponsor card that uses props to populate the display information. (I did not add the w-9 label because it was not in the developer figma view)
- `/public/sponsor_card_icon _mail_.png`: Added mail icon for the sponsor card
- `/public/sponsor_card_icon _user_.png`: Added user icon for the sponsor card
- `/public/sponsor_profile_picture_placeholder.png`: Added profile picture placeholder for the sponsor card

### Testing Considerations

Tested locally

### Pull Request Checklist

- [x] Code is neat, readable, and works
- [x] Comments are appropriate
- [x] The commit messages follows our [guidelines](https://h4i.notion.site/Conventional-Commits-593452ad1179489399ad3bd696ef772a)
- [x] The developer name is specified
- [x] The summary is completed
- [x] Assign reviewers

### Screenshots/Screencast

image

### CODE REVIEW

File: src/components/sponsor_card.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
+import Image from "next/image";
+
+type sponsor_card_props = {
```
Comment (ryanhu021) [inline src/components/sponsor_card.tsx]: could we change this to an interface and rename this to `SponsorCardProps` to keep prop conventions consistent?

**Review by ryanhu021 — CHANGES_REQUESTED** [review]
one small thing, everything else looks good!

**Comment by ryanhu021** [comment]
also could you move the images to `public/images`?

File: src/components/sponsor_card.tsx (hunk 1 above)
Comment (ktaschek) [reply src/components/sponsor_card.tsx]: 👍

**Review by ryanhu021 — APPROVED** [review]
Looks good!

**Pull request merged**